### PR TITLE
Add Commands for other Modules

### PR DESCRIPTION
**Pull request by Youri970410**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 [hunk 1]
```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
...
-    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}}
+    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}},
+    @{Name = "Az.Accounts";                   Command = {Get-AzDomain -ErrorAction Stop}},
+    @{Name = "Az.Advisor";                    Command = {Get-AzAdvisorConfiguration -ErrorAction Stop}},
+    @{Name = "Az.DataBoxEdge";                Command = {try {Get-AzDataBoxEdgeBandwidthSchedule -ResourceGroupName $resourceGroupName -DeviceName 'smokeTestDevice' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.Databricks";                 Command = {try {Get-AzDatabricksVNetPeering -ResourceGroupName $resourceGroupName -WorkSpaceName 'smokeTestWorkSpace' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.DataShare";                  Command = {try {Get-AzDataShare -ResourceGroupName $resourceGroupName -AccountName 'smokeTestAccountName' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*NotFound*") {throw $_}}}},
```
Comment (BethanyZhou) [inline tools/Test/SmokeTest/RmCoreSmokeTests.ps1:92]: @Youri970410 Could you check the exported cmdlets inside each added module and find the simplest one (requires the least parameter) for smoke test

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 [hunk 2]
```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
...
+    #The test for Az.DeploymentManager has done,but it has one issue when deserializing the response
+    #@{Name = "Az.DeploymentManager";          Command = {try {Get-AzDeploymentManagerArtifactSource -ResourceGroupName $resourceGroupName  -ErrorAction Stop}catch {if ($_.ToString() -notlike "*not found*") {throw $_}}}},
```
Comment (BethanyZhou) [inline tools/Test/SmokeTest/RmCoreSmokeTests.ps1:94]:
```suggestion
    # Waiting for an issue fix: https://github.com/Azure/azure-powershell/issues/13522#issuecomment-728659457
    # @{Name = "Az.DeploymentManager";          Command = {try {Get-AzDeploymentManagerArtifactSource -ResourceGroupName $resourceGroupName  -ErrorAction Stop}catch {if ($_.ToString() -notlike "*not found*") {throw $_}}}},
```

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 [hunk 3]
```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
...
+    @{Name = "Az.PrivateDns";                 Command = {try {Get-AzPrivateDnsRecordSet -ResourceGroupName $resourceGroupName -ZoneName 'smokeTestZone' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.SqlVirtualMachine";          Command = {try {Get-AzAvailabilityGroupListener -ResourceGroupName $resourceGroupName -SqlVMGroupName 'smokeTestSqlVMGroup' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.StorageSync";                Command = {try {Get-AzStorageSyncCloudEndpoint -ResourceGroupName $resourceGroupName -StorageSyncServiceName 'smokeTestStorage' -SyncGroupName 'smokeTestSyncGroup' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
+    @{Name = "Az.Support";                    Command = {try {Get-AzSupportProblemClassification -ServiceId '55d5fb0b-eae8-45f0-8182-28119dcc12fa' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}}
```
Comment (BethanyZhou) [inline tools/Test/SmokeTest/RmCoreSmokeTests.ps1:106]: ServiceId 00000000-0000-0000-0000-000000000000 is fine

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 [hunk 4]
```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
     @{Name = "Az.Resources [Tags]";           Command = {Get-AzureRmTag -ErrorAction Stop}},
     @{Name = "Az.TrafficManager";             Command = {Get-AzureRmTrafficManagerProfile -ErrorAction Stop}},
     @{Name = "Az.Billing [UsageAggregates]";  Command = {Get-UsageAggregates -ReportedStartTime '1/1/2018' -ReportedEndTime '1/2/2018' -ErrorAction Stop}},
-    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}}
+    @{Name = "Az.Websites";                   Command = {Get-AzureRmWebApp -ErrorAction Stop}},
+    @{Name = "Az.Accounts";                   Command = {Get-AzDomain -ErrorAction Stop}},
+    @{Name = "Az.Advisor";                    Command = {Get-AzAdvisorConfiguration -ErrorAction Stop}},
+    @{Name = "Az.DataBoxEdge";                Command = {try {Get-AzDataBoxEdgeBandwidthSchedule -ResourceGroupName $resourceGroupName -DeviceName 'smokeTestDevice' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply tools/Test/SmokeTest/RmCoreSmokeTests.ps1]: Got it,Thanks.

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 [hunk 5]
```diff
@@ -84,7 +84,26 @@ $resourceTestCommands = @(
...
+    @{Name = "Az.Databricks";                 Command = {try {Get-AzDatabricksVNetPeering -ResourceGroupName $resourceGroupName -WorkSpaceName 'smokeTestWorkSpace' -ErrorAction Stop}catch {if ($_.ToString() -notlike "*Not Found*") {throw $_}}}},
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply tools/Test/SmokeTest/RmCoreSmokeTests.ps1]: Got it,Thanks.

File: tools/Test/SmokeTest/RmCoreSmokeTests.ps1 (hunk 1 above)
Comment (Youri970410) [reply tools/Test/SmokeTest/RmCoreSmokeTests.ps1]: Got it,Thanks for your reminder.

**Comment by BethanyZhou** [comment]
New smoke test script is tested on Azure DevOps: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=618654&view=results. LGTM

**Review by BethanyZhou — APPROVED** [review]

**Comment by BethanyZhou** [comment]
Successfully re-tested after polish codes: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=622396&view=results

**Pull request merged**